### ISSUE TITLE
 Issue when i press the "add definition" button in the card browser

### ISSUE THREAD

**Issue by IEatRamen234** [Ajatt-Tools/Japanese#15]
1) OS name: Linux mint
2) OS version: 21.3 Cinnamon
3) Anki version: 24.06.2, installed from script. I also encountered the problem in older versions of anki, both flatpak and script.
When i press the "add definition" button, i get this error: "Connection Error". I have it set up correctly in the settings, as seen in the image
![image](https://github.com/Ajatt-Tools/Japanese/assets/171134790/6c2d0668-5d5e-4c2d-856d-91fb5a818320)
![image](https://github.com/Ajatt-Tools/Japanese/assets/171134790/3009aa3d-9610-4eeb-9d1c-b10a038e7aad)

**Comment by tatsumoto-ren** [comment]
The "add definition" button works by fetching definitions from https://sakura-paris.org. See [this file](https://github.com/Ajatt-Tools/Japanese/blob/v24.5.18.0/helpers/sakura_client.py) for details. 

Sakura-paris was an online dictionary website. But it died a while ago. There's ongoing work to replace it, and it will be implemented in future versions of AJT Japanese.

**Comment by IEatRamen234** [comment]
Thanks! now i know that the issue is taken care of. good luck!!

**Comment by tatsumoto-ren** [comment]
While the feature is not ready yet, in the mean time you can add definitions using either [Rikaitan](https://tatsumoto.neocities.org/blog/setting-up-yomichan) or [Goldendict](https://tatsumoto.neocities.org/blog/setting-up-goldendict).

**Comment by Furshia** [comment]
Would it be possible to have custom sources for the "add definition"?

**Comment by tatsumoto-ren** [comment]
Hopefully, yes.

**Comment by PANICBUTT0N** [comment]
would jisho work? there are some libraries designed exclusively to act as an api for it.

**Comment by Hundway** [comment]
I created [Waydict](https://github.com/Hundway/anki-way-dict), an Anki add-on to import definitions from dictionaries in Yomitan format. It could be useful.

**Comment by tatsumoto-ren** [comment]
@Hundway I'll take a look. We could probably integrate that into AJT Japanese.

**Comment by Hundway** [comment]
@tatsumoto-ren Sure. Let me know if you need any help.

**Comment by IEatRamen234** [comment]
have u also implemented bulk add feature? i did not have the time to check the addon properly but i cant seem to find it... thanks anyway for your great work!

**Comment by Hundway** [comment]
Yes, you can select multiple cards and use the "Waydict: Add definition" option in the right-click context menu to bulk add definitions.

**Comment by Tsogtpurev** [comment]
Hello Tatsumoto-san add definition not working it says connection error do you know why? Can you help me thanks

**Comment by tatsumoto-ren** [comment]
@Tsogtpurev the website that hosted the dictionaries got taken down. We will introduce a different feature where definitions will be looked up in Rikaitan dictionaries.
